microsoft-edge/devtools-guide-chromium/javascript/consume-source-maps-from-azure.md

```diff
@@ -26,13 +26,53 @@ To learn about how to publish source maps, see [Securely debug original code by
 
 
 <!-- ====================================================================== -->
-## Step 1: Generate a Personal Access Token for Azure DevOps
+## Step 1: Enable source maps in DevTools
 
-To obtain source maps from the Azure Artifacts symbol server, DevTools uses the fetch protocol to talk to Azure DevOps, which requires a valid Personal Access Token (PAT).
+To make sure that source maps are enabled in DevTools:
 
-You need to generate a PAT even if you already generated one to publish source maps to the symbol server in [Securely debug original code by publishing source maps to the Azure Artifacts symbol server](publish-source-maps-to-azure.md).
+1. To open DevTools, in Microsoft Edge, right-click a webpage, and then select **Inspect**.  Or, press **Ctrl+Shift+I** (Windows, Linux) or **Command+Option+I** (macOS).
+
+1. In DevTools, click **Settings** (![Settings icon](./consume-source-maps-from-azure-images/settings-gear-icon-light-theme.png)) > **Preferences**.
+
+1. Make sure the **Enable JavaScript source maps** checkbox and the **Enable CSS source maps** checkbox are selected:
+
+    ![The Preferences configuration screen in DevTools settings](./consume-source-maps-from-azure-images/preferences-source-maps-devtools.png)
+
+
+<!-- ====================================================================== -->
+## Step 2 - Connect DevTools to the Azure Artifacts symbol server
```

Comment:
```suggestion
## Step 2: Connect DevTools to the Azure Artifacts symbol server
```
like Step 1.  Global in article: check headings "Step" punctuation.  Colon is more standard punctuation usage (though hyphen might be more "visible")